perf/stability/mysql/templates/server.yaml

```diff
@@ -0,0 +1,75 @@
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: mysql-pvc
+spec:
+  accessModes:
+    - ReadWriteMany
+  volumeMode: Filesystem
+  resources:
+    requests:
+      storage: 1Gi
+  storageClassName: "fast"
```

Comment:
Not familiar with PVCs - do we have to define some "fast" storage type? Does this work on all clouds, minikube, etc?

Reply:
This is a valid concern. I consult someone from k8s storage side. And got suggestion to use a more up-to-date/accurate sample YAML.

Will do and ping here once it's done.

Reply:
Done. grab yaml sample without storageclass. The cluster will use the default one.